app/controllers/publications_controller.rb

```diff
@@ -57,18 +57,19 @@ def special_params
   end
 
   def publications_query_string
-    level_one_taxon = params['taxons'].try(:first) || params['topics'].try(:first)
-    level_two_taxon = params['subtaxons'].try(:first)
+    allowed_params = cleaned_document_filter_params
```

Comment:
Can this be called `filters_params` - or `filters_params` called `allowed_params`?

Reply:
I shall make it so.